index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Kareem Wilson</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <ul class="nav-list">
+        <li class="nav-item"><a href="#Welcome">Portofolio</a></li>
+        <li class="nav-item"><a href="#About">About</a></li>
+        <li class="nav-item"><a href="#Contact">Contact</a></li>
+        <li class="fa fa-bars"></li>
+      </ul>
+      <div class="content">
+        <section id="Welcome" class="welcome-section">
+          <h1 class="title">Hello there, I'm Kareem.</h1>
+          <h2 class="subTitle">I'm a Software Developer</h2>
+          <h4 class="bio">
+            Microuant in Microverse, My favourite hobby turning coffee and
+            legacy code into clean code, Looking forward to play with the chips
+            of Computers!
+          </h4>
```

Comment:
Please, use a paragraph (`<p>`) tag for the bio. This way will be semantically correct and the screen readers and other technologies will thank you.